content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,153 @@
+page][community] which includes links to our Slack channel and community meetings.
+
+[ingc]:https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class
+[strc]:https://kubernetes.io/docs/concepts/storage/storage-classes/
+[crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
+[concepts]:https://gateway-api.sigs.k8s.io/concepts/api-overview/
+[guides]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+[install-crds]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/#install-the-crds
+[issue]:https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[disc]:https://github.com/kubernetes-sigs/gateway-api/discussions/new
+[community]:https://gateway-api.sigs.k8s.io/contributing/community/
+
+## Release highlights
+
+### Graduation to beta
+
+The `v0.5.0` release is particularly historic because it marks the growth in
+maturity to a `v1beta1` release for some of our key APIs:
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
+
+- API has been [widely implemented][impl].
+- Conformance tests provide basic coverage for all resources and have multiple implementations passing tests.
+- Most of the API surface is actively being used.
+- Kubernetes SIG Network API reviewers have approved graduation to beta.
+
+To see what's in store for future releases check out the next steps section.
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Release channels
+
+This release introduces the `experimental` and `standard` [release channels][ch]
+which enable a better balance of maintaining stability while still enabling
+experimentation and iterative development.
+
+The `standard` release channel includes:
+
+- resources that have graduated to beta
+- fields that have graduated to standard (no longer considered experimental)
+
+The `experimental` release channel includes everything in the `standard` release
+channel, plus:
+
+- `alpha` API resources
+- fields that are considered experimental and have not graduated to `standard` channel
+
+Release channels are used internally to enable iterative development with
+quick turnaround, and externally to indicate feature stability to implementors
+and end-users.
+
+For this release we've added the following experimental features:
+
+- [Routes can attach to Gateways by specifying port numbers](https://gateway-api.sigs.k8s.io/geps/gep-957/)
+- [URL rewrites and path redirects](https://gateway-api.sigs.k8s.io/geps/gep-726/)
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 release page][releasepage].
+
+[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3
```

Comment:
```suggestion
[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.5.0
```
BTW for this site we prefer inline links rather than the footnote style, but that's a preference not a merge blocker.